src/Init/Tactics.lean

```diff
@@ -1280,6 +1280,12 @@ a lemma from the list until it gets stuck.
 syntax (name := applyRules) "apply_rules" (config)? (&" only")? (args)? (using_)? : tactic
 end SolveByElim
 
+/-- Syntax for `exact?` -/
```

Comment:
It would be nice if this docstring described what `exact?` does! What about something like:
```suggestion
/-- Searches all libraries in scope for a definition that can solve the goal using `exact` -/
```

What does the `using` option do here?